package.json

```diff
@@ -16,7 +17,9 @@
     "build": "npm run clean && npm run build:code && npm run build:flow",
     "clean": "rimraf dist",
     "typecheck:flow": "flow check --max-warnings=0",
+    "typecheck:ts": "dtslint types",
     "lint": "eslint src test",
+    "lint:ts": "tslint types",
```

Comment:
Do we really need a linter for definitions other then `dtslint`?

Reply:
actually - no, we dont need it.